docs/changes.rst

```diff
@@ -45,6 +45,24 @@ details of the changes themselves and the reasons for the changes.
 
 * Removed the `verify_callback` parameter of `WBEMConnection`.
 
+* Removed the deprecated ability to set the following properties of class
+  `WBEMConnection`: `url`, `creds`, `x509`, `ca-certs`, `no_verification`,
+  and `timeout`.
+
+* Removed the deprecated methods `method_call()` and imethod_call()` and the
+  deprecated property `operation_recorder` from class `WBEMConnection`.
+
+* Removed the deprecated property `property_list` from class `CIMInstance`.
```

Comment:
add  "and constructor argument"

Reply:
DONE - added "and the same-named init parameter".